draft-ietf-trans-rfc6962-bis.md

```diff
@@ -950,10 +949,12 @@ extension that it does not understand, it SHOULD ignore that extension.
 Furthermore, an implementation MAY choose to ignore any extension(s) that it
 does understand.
 
-The encoding of the digitally-signed element is defined in [RFC5246].
+`signature` is computed using the signature algorithm declared in the log's
+parameters (see {{log_parameters}}).
 
-`timestamped_entry` is a `TransItem` structure that MUST be of type
-`x509_entry_v2` or `precert_entry_v2` (see {{tree_leaves}}).
+The input to the signature is a `TransItem` structure
+that MUST be of type `x509_entry_v2` or `precert_entry_v2` (see
+{{tree_leaves}}).
```

Comment:
I suggest combining this paragraph and the previous paragraph as follows:

`signature` is computed over a `TransItem` structure of type `x509_entry_v2` or `precert_entry_v2` (see {{tree_leaves}}) using the signature algorithm declared in the log's parameters (see {{log_parameters}}).